samples/ConPTY/EchoCon/EchoCon/EchoCon.cpp

```diff
@@ -0,0 +1,207 @@
+// EchoCon.cpp : Entry point for the EchoCon sample application.
+//
+
+#include "stdafx.h"
+#include <Windows.h>
+#include <conio.h>
+#include <process.h>
+#include <wchar.h>
+
+void __cdecl PipeListener(LPVOID);
+HRESULT InitializeStartupInfoAttachedToConPTY(STARTUPINFOEX*, HPCON);
+COORD GetConsoleSize(HANDLE);
+
+int main()
+{
+    TCHAR szCommand[] = L"ping localhost";
```

Comment:
you used `TCAHR` and `L` together: big no-no! either standardize on `W` APIs everywhere and eat the bullet with `wchar_t`, or don't commit this :)

Reply:
Fair. Fixed.